hack/package/rpm/build.sh

```diff
@@ -41,7 +41,7 @@ function build_lxcfs ()
     git clone -b "$LXC_BRANCH" https://github.com/lxc/lxcfs.git && cd lxcfs
     
     # change liblxcfs.so to libpouchlxcfs.so
-    sed -i 's/liblxcfs/libpouchlxcfs/g' "$(grep -r "liblxcfs" . |awk -F':' '{print $1}'|uniq )"
+    grep -r "liblxcfs" . |awk -F':' '{print $1}'|uniq | xargs sed -i 's/liblxcfs/libpouchlxcfs/g'
```

Comment:
we can use `grep -l -r "liblxcfs" . | xagrs sed -i 's/liblxcfs/libpouchlxcfs/g'`. 😄 WDYT?

Reply:
GOOD!